pkg/serverless/trace/propagation/carriers_test.go

```diff
@@ -169,22 +169,22 @@ func TestSnsSqsMessageCarrier(t *testing.T) {
 				}`,
 			},
 			expMap: nil,
-			expErr: errors.New("Error unmarshaling message body: json: cannot unmarshal string into Go struct field .MessageAttributes of type map[string]struct { Type string; Value string }"),
+			expErr: errors.New("Error unmarshaling message body: json: cannot unmarshal string into Go struct field SNSEntity.MessageAttributes of type map[string]interface {}"),
```

Comment:
I don't normally like asserting error text like this, but doing so makes sure that we are actually hitting the expected error, ensuring 100% test coverage.